### DISCUSSION TITLE
CRON- Scheduled activity [Aug 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3294] [Discord Threads]
> 🙋‍♂️ **AlexU**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### No, not at the moment.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1008203884239523850)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
